Core/Services/StitchCoreRemoteMongoDBService/Tests/StitchCoreRemoteMongoDBServiceTests/Sync/NamespaceSynchronizationConfigTests.swift

```diff
@@ -35,7 +35,7 @@ class NamespaceSynchronizationConfigTests: XCMongoMobileTestCase {
 
         docConfig = CoreDocumentSynchronization.init(docsColl: docsColl,
                                                      namespace: namespace,
-                                                     documentId: documentId.bsonValue,
+                                                     documentId: AnyBSONValue(documentId),
```

Comment:
They shouldn't have to, but it doesn't matter. They may have to use it when using the Swift driver regardless. `AnyBSONValue` is there so that BSONValue's can generically be Codable. It's intentionally exposed.